src/S3ClientGUI/s3treemodel.cpp

```diff
@@ -143,10 +182,10 @@ void S3TreeModel::setRootPath(const QString &path) {
 
     qDebug() << "prefix name" << prefix;
 
-	//in begining, insert ".." Folder
-	QVariantList data;
+	//in begining, insert ".." Folder---move to listObjectFinished()
```

Comment:
If you want to remove some code, just REMOVE it ,  do not comment  the code

Reply:
I konw